src/lib/pickles/cache_handle.mli

```diff
@@ -1,4 +1,4 @@
-(* Cache handle *)
+(** Cache handle. It is currently used to cache proving and verifying keys for pickles *)
```

Comment:
FWIW: the double star `**` is not necessary for the first doc-comment. It used to be the case (if I'm not mistaken) that the first comment needed to be a single star to be used in the generated HTML!

Reply:
Ah, I didn't know.